recipes/volk/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "1.2.198.0":
+    url: "https://github.com/zeux/volk/archive/1.2.198.tar.gz"
+    sha256: "8f3869d94ccce265eaef064b1be0af2d8de925e512d3413c1cf02591b99597e2"
```

Comment:
why the additional `.0` in the version number ?
also, [1.2.198](https://github.com/zeux/volk/releases/tag/1.2.198) tag is actually the commit 713ab8bfcd9d8e5fe78ccf85806c0b4e5c566f71, which is used for version 1.2.195 below. Why is that ?

Reply:
volk is updated not for each minor version of vulkan-headers.
Recipe is implemented the way, it's version should match vulkan-headers version. There is only '1.2.198.0' version and no '1.2.198' version

Reply:
I would drop the `.0`.
I [can't see the suffix](https://github.com/zeux/volk/releases/tag/1.2.190).

You say the volk version number must match the vulkan version.
How come the `.0` suffix was not needed before?

Reply:
You can add `vulkan-headers/1.2.198`, which is exactly the same commit than `vulkan-headers/1.2.198.0` already packaged.

Reply:
If we have both vulkan-headers/1.2.198 and vulkan-headers/1.2.198.0, why can't we have two versions of volk then?